.eslintrc.json

```diff
@@ -1,10 +1,7 @@
 {
     "root": true,
     "parser": "@typescript-eslint/parser",
-    "plugins": [
-        "@typescript-eslint",
-        "react-hooks"
-    ],
+    "plugins": ["@typescript-eslint", "react-hooks"],
```

Comment:
can we revert the changes to this file?

Reply:
it is just linting, it automatically made it one line when I was enabling/disabling react-hooks/exhaustive-deps and save the file. Will keep it as is